### PR TITLE
Atlas-class Light Armored Crusier

### PR DESCRIPTION
**Pull request by meemofcourse**
## About The Pull Request
imagen
atlas2

The Atlas-class is the first Colonial League ship following the CLIP Rework. The decaling and general decoration of the ship needs a revision, but otherwise, it is intended as a small, overall generalist patrol vessel to the likes of the Vaquero. The 4-Minutemen team, lead by one Sergeant, is subordinate to the First Officer and Captain, who overlook the operation of the ship by the Corpsman and the Engineer.

It is staffed by:
* 1 Captain
* 1 First Officer
* 1 Sergeant
* 1 Corpsman
* 1 Engineer
* 4 Minutemen
* 1 Deckhand

## Why It's Good For The Game
it's the clip ship it uses the colon league content

## Changelog

:cl:
add: Atlas-class Light Armored Crusier
/:cl:

### CODE REVIEW

**Comment by kaylexis** [comment]
CLIP IS SO BACK.

**Comment by Martinpachu** [comment]
it's peak chat

**Comment by Martinpachu** [comment]
imagen
it has a 762 box but no guns that use 762 i think?

**Comment by Martinpachu** [comment]
there's no gravity generator!!

**Comment by Martinpachu** [comment]
i might be a stupid baby but i don't think the cargo bay doors are linked to any buttons. also no holofields ? also just my dumb gay opinion but there could be another candor mag

**Comment by firebudgy** [comment]
image
I think you forgot a couple wires for the engines here

**Comment by firebudgy** [comment]
image
Bridge lockdown button closes all these shutters, while the bridge windows button does nothing

image
Every other ship usually has windows blocking the dorms so SSD people don't get obliterated by carps while away from their computer, so I'm just wanting point it out in case you forgot or something

image
Same with the medical bay external window shutters, I'm just pointing this out in case their omission was an accident

**Comment by Sadhorizon** [comment]
- They don't have a single space suit on the entire ship.
obraz
- Why is it just... sticking out like that... _surely_ there is a better way to handle it. Maybe a window into grilles?
obraz
- Most ships have shutters on the precharger. Cuz, technically, you are blocking engine output rn. (lore-wise)
- The cargo bay has very little cargo space. I'd suggest, on the right side, making more room for a crate shelf. Also switching the air tank rack for an air tank crate, cuz it's less stationary.
obraz
- This locker has the wrong dir.
obraz
- This airlock has no controller and no vents.
obraz
- Empty space spotted. Could put a vendor or something here. Cigs never hurt anybody.
obraz
- No air pump? Also, missing cable (marked green).
obraz
- You forgot holofields, and also a shutter button.
- Bridge airlocks don't have access req set.
- Airlocks aren't named. Probably not needed, but still nice to have.
obraz
- You have two air cans, but no ports to put them in. Would probably be nice.
- All of your light switches are like. Very high. Latency explained to me before, that shadow should be touching the ground. Just pointing it out.
obraz
- When those are next to each other, .i makes you speak on both, which is kinda annoying.
- Medical lacks surgical tools.
- There is no CLIP stamp on board.

**Comment by Martinpachu** [comment]
small suggestion, maybe start with a bit of glass and iron for the autolathe?

**Comment by Martinpachu** [comment]
imagen
the turret panel is unreachable

**Comment by Sadhorizon** [comment]
> small suggestion, maybe start with a bit of glass and iron for the autolathe?

Some plastic is neat to have too! Just a lil bit, for when you need it.

**Comment by Martinpachu** [comment]
oh, could you add in a fuel and water tanks too? for extinguishers and welding tools.

**Comment by Martinpachu** [comment]
deleted because im stupig nevm

**Comment by meemofcourse** [comment]
> imagen it has a 762 box but no guns that use 762 i think?

imagen
:(

**Comment by meemofcourse** [comment]
> Why is it just... sticking out like that... surely there is a better way to handle it. Maybe a window into grilles?

nah
> Empty space spotted. Could put a vendor or something here. Cigs never hurt anybody.

i am avoiding vendors on the argument that this is a military vessel
> There is no CLIP stamp on board.

intended for admin use. you shouldn't be able to stamp things on behalf of the entire country

**Comment by Martinpachu** [comment]
imagen
cargo bay doors opens only one door. also, idk if i just didn't see it, but there should be a cell charger somewhere. since that's standard for every ship i believe.

**Comment by Martinpachu** [comment]
actually, weren't communications consoles off limits? since you can do stuff like request nuclear access codes and send announecements. i vaguely remember them not being allowed.

**Comment by Martinpachu** [comment]
is the sargeant supposed to go out with the minutemen, as like leading an away squad? if so, they should probably have at least a handgun for themselves.

**Comment by Martinpachu** [comment]
sorry for not saying all this stuff all at once by the way. i just realize some stuff later

**Review by goober3 — DISMISSED** [review]
hi, i'm 'Shiptest Maptainers' and i am going to review this PR.

f1f64e81897216c2086df5c143c19bd9
the cryo room could do with a light switch.

8f7580882b9d75c13bef97899ad81a54
consider removing the curtain here, i don't think it works with the cyborg recharger sprite.

9ec48364337401afc8db399a08ebea88
please make these sidings reach the bottom of the room i beg

d479ab47eda7ea6bfbefd70f70e70fc9
these railings don't look great in my opinion!!! consider making it a single rail or removing them altogether. also look into accenting the floor in the wider canteen area with some techfloor to make it a little more visually interesting.

659420d0ee230553c0f6d3bf1b090e5f
i think the cell here warps the ship shape a little too much. could you maybe move it to where the grilles are here, or somewhere else on the ship.
image

ed3e8b83d90f425fcb1fa1a2875042b8
i like this hallway. why is there a microwave here. can you put warning stripes on the firelocks and replace them with directional variants by the way.

ff6c5492a1bbd39a9dd05fca8f034920
i also really like this room. consider giving them a crate shelf (x1)

image
oh no, i guess i can't perform this life-saving surgery. what a shame.

we have CLIP guns now. use them. look into [code/modules/projectiles/guns/manufacturer/clip_lanchester/ballistics.dm](https://github.com/shiptest-ss13/Shiptest/pull/3238/files#diff-4ef8305cfcce0303b5abd84ec7df5d12446c01acb9d5aa697fdd5c139234e6e4) for an idea of what to use where. (replace the captain's pistol with a CM-70)

74f62c97f4ea57dbb6c7c5d19cf03eae
guess i'll fly into an asteroid field then.

image
why does the first officer have so much clothing

e7880e40a6e8f5b207444db7eba0afc4
consider more catwalks and rusted plating on both sides of the engine room. this seems a little too clean and orderly. also, add windoors to the engine chargers.

6ea8fe7f53523cf87cc90c18210d145c
these shutters (bottom) appear to be connected to the dorm windows rather than the FO's privacy shutters

that's about it i think

**Comment by meemofcourse** [comment]
> consider removing the curtain here, i don't think it works with the cyborg recharger sprite.

but lat did it in the panacea and nobody said anything :(
> can you put warning stripes on the firelocks and replace them with directional variants by the way.

yes, but i'm not changing them to directionals because this is a door dividing a hallway and the thin ones work best for airlocks
> oh no, i guess i can't perform this life-saving surgery. what a shame.

> guess i'll fly into an asteroid field then.

cdn is fucked what do you want me to do
> why does the first officer have so much clothing

formal uniforms to hand to the crew. you know, it's a military

**Comment by rye-rice** [comment]
I will assume by lack of maptainer review this is ready to merge

**Comment by meemofcourse** [comment]
please don't until another maptainer approves it

**Comment by rye-rice** [comment]
@Latentish @Apogee-dev

File: _maps/configs/minutemen_atlas.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+	{
```
Comment (Sadhorizon) [inline _maps/configs/minutemen_atlas.json:1]: Add `"faction": "/datum/faction/clip",` somewhere, or it's gonna show as indie on the spawning menu.

**Review by Gristlebee — COMMENTED** [review]
The table in the engineering wing merges together under the railing.
image
image
Replace the regular box of shotgun shots with an ammo box please.
image
Could you also put the double barrel and CM-23s the officers get into gun cases for consistency?

This is a strong arsenal to be starting with on roundstart. I'd suggest cutting a CM-5s and CM-82s and replacing it with two CM-23 pistols.
image

This is funny, but I'm concerned about them not having a functioning microwave with comparatively few starting rations to the ship's crew size. Maybe put a microwave board in engineering they can put together, or give them some more rations.
image

I'd also suggest cutting one of the 20 stacks of plasma in the engine room. The ship doesn't need that much fuel to start with I feel.

**Comment by DIB-DOG** [comment]
Having some survival knives or any melee weapon really, would be nice for the minutmen as well.

**Review by FalloutFalcon — APPROVED** [review]

**Review by Gristlebee — COMMENTED** [review]
Please reduce the amount of extra mags for the CM-82, and CM-5 to two since there's only one of each now. It would also be good to swap out the combat knives for survival knifes to bring it in line with other ships.
image

Aside from that, everything looks pretty good to me.

**Comment by Erikafox** [comment]
recently played on this. here are my thoughts

image
you should curve the railings, the middle tile that's obviously technology feels weird to stand on. 
image
Honestly thought this was kinda ugly
image
don't care for the engines in this style. I think being able to see the terminals is kinda assumed. Maybe put some decalling where they should be? The steeldecal plates are good for that.
First Officer and Captain should have a spare CMM bag (and imo should spawn with one, the captain bag doesn't fit them too well)
image
This feels somewhat haphazard
Airlocks all need to be labelled
image
this is like. a lot of unused space imo. take some of it and give the captain a nice little front office?
image
spread some folders out on the tables - and put some paperwork stuff in the bridge too.
image
this does not look good in game
image
You may want to secure the blackbox somewhat
image
These should be SSU'd. maybe take some cargo bay space (and expand it a bit) to accomodate, or eat into the briefing area.
image
grav gen feels kinda just. tacked on there.

**Comment by meemofcourse** [comment]
thats it bitch im coming for you

**Comment by meemofcourse** [comment]
> don't care for the engines in this style

What

> take some of it and give the captain a nice little front office?

no

> put some paperwork stuff in the bridge too.

there is a paper cutter, paper bin, photocopier, clipboard, and stamps i literally dont know what more you want. i'm not adding the clip stamp because standard individual ships should not be able to give the approval of an entire nation

**Comment by Erikafox** [comment]
>engines

I just prefer the style of seeing the terminal and having more like.. technical grunge around it

Rest is fair its your ship

**Comment by Sadhorizon** [comment]
please fix this
obraz

**Comment by goober3** [comment]
a8d2c65a1b1debfea67dcc6eb6715715
just a thought- consider turning this external injector into more of a feature somehow. maybe some decals or railings, think about expanding the platform its on. it feels somewhat haphazard in its placement at the moment. also please fill that corner next to the engine precharger or move the water tank towards it.

**Comment by Gristlebee** [comment]
This PR should be good to go.

**Pull request merged**